playbooks/roles/capsule/tasks/install.yml

```diff
@@ -47,6 +47,11 @@
     src: "/tmp/{{ inventory_hostname }}{{ capsule_certs_tar }}"
     dest: ~/
 
+- name: 'Change cert permissions'
+  shell: chmod 755 -R /etc/pki/katello/private
+  delegate_to: "{{ capsule_server }}"
+  when: devel == true
```

Comment:
Just something I have had to do in the dev environment for a successful capsule install